app/controllers/api/v1/concerns/content_type.rb

```diff
@@ -8,9 +8,15 @@ module ContentType
           before_action :validate_content_type!
         end
 
-        # force all requests to be application/json
+        # default to JSON responses, disallow other Accept content types or format requests
+        # will allow */* or application/json in any part of Accept header and respond with JSON
         def validate_content_type!
-          head 406 unless request.headers['Accept'] === 'application/json'
+          accept_header = request.headers['Accept'].present? ? request.headers['Accept'] : ''
```

Comment:
I've seen conditions like this condensed to ` if !accept_header.match(Regexp.union(['*/*', 'application/json'])) `

I don't feel strongly about whether that's more/less readable here.

Reply:
I like it - the pithier the better.  And I actually think that's just as readable.